content/applications/sales/subscriptions.rst

```diff
@@ -31,49 +75,49 @@ Subscription quotations
 .. important::
    Sales orders with a defined recurrence become subscriptions.
 
-To create a new subscription, click on :guilabel:`New` from the *Subscription* or the :doc:`Sales
-</applications/sales>` app. You can either:
+To create a new subscription, click on :guilabel:`New` from the **Subscriptions** or the :doc:`Sales
+</applications/sales>` app. Then, from the resulting quotation, either:
 
-- Select a :doc:`subscription plan </applications/sales/subscriptions/plans>` to prefill the
-  quotation instantly, or
-- Fill out the quotation normally, making sure to select a recurrence and an end date if necessary
-  and adding :doc:`recurrent products </applications/sales/subscriptions/products>`.
+- Select a :doc:`subscription plan </applications/sales/subscriptions/plans>` to auto-populate the
+  quotation instantly.
+- Or, fill out the quotation normally, making sure to select a recurrence, an end date (if
+  necessary), and add :doc:`recurrent products </applications/sales/subscriptions/products>`.
 
 .. tip::
-   You can define different invoice and delivery addresses by enabling the :doc:`Customer Addresses
+   Define different invoice and delivery addresses by enabling the :doc:`Customer Addresses
    </applications/finance/accounting/customer_invoices/customer_addresses>` feature.
 
 .. _subscriptions/confirmation:
 
 Confirmation
 ============
 
-Send the quotation to the customer for confirmation by clicking on :guilabel:`Send by email`, or
-confirm it immediately by clicking on :guilabel:`Confirm`.
+Send the quotation to the customer for confirmation by clicking :guilabel:`Send by Email`, or
+confirm it immediately by clicking :guilabel:`Confirm`.
 
 .. tip::
-   Click on :guilabel:`Customer Preview` to preview the customer portal where the customer can view
-   their quotation, sign and pay it, and communicate with you.
+   Click :guilabel:`Customer Preview` to preview the customer's point-of-view their quotation. From
+   there, customers can sign and pay it, and communicate with the company, if needed.
 
 .. _subscriptions/automatic-payments:
 
 Automatic payments
 ==================
 
-You can require the customer to set an automatic payment method and pre-pay the subscription's first
+Require the customer to set an automatic payment method, and pre-pay the subscription's first
```

Comment:
```suggestion
Prompt the customer to set up an automatic payment method and pre-pay the subscription's first
```

Reply:
Optional/stylistic: I've not seen "require" used much in the imperative so this gave me pause. Suggesting an alternative here, feel free to use or discard.